include/deal.II/grid/tria_accessor.h

```diff
@@ -1605,6 +1606,12 @@ class TriaAccessor : public TriaAccessorBase<structdim, dim, spacedim>
   is_translation_of(
     const TriaIterator<TriaAccessor<structdim, dim, spacedim>> &o) const;
 
+  /**
+   * Geometric entity type of the current object.
+   */
+  inline GeometricEntityType
+  geometric_entity_type() const;
```

Comment:
No need to put `inline` in front of the declaration. It's only necessary with the definition.